provision.sh

```diff
@@ -27,6 +27,12 @@ wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
 echo "--------------- Installing both virtualenv and virtualenvwrapper ---"
 pip install virtualenvwrapper virtualenv
 
+# install postgresql and setup user
+echo "--------------- Installing postgresql ------------------------------"
+apt-get install -y python-dev libpq-dev postgresql postgresql-contrib
+su - postgres -c "createuser -s vagrant"
+su - vagrant -c "createdb $DB"
```

Comment:
@alayek so should I use ${variable} all the time? Is there any reason why one should use $variable at any time? Why are there these two different versions though if one is more specific anyways?